src/utils/ethereum.js

```diff
@@ -80,6 +80,29 @@ export async function checkEthereumProvider() {
   return { networkId, address };
 }
 
+export function browserEthereumProviderAddress() {
```

Comment:
note that depending on `selectedAddress` may leave out some clients. check out makerdao/migrate-dashboard#13, where I switched to using `getAccounts`; if we could do the same thing here that would be good.